runtime/jailer_test.go

```diff
@@ -96,3 +100,39 @@ func TestJailer_invalidUIDGID(t *testing.T) {
 	_, err := newJailer(context.Background(), logrus.NewEntry(logrus.New()), "/foo", &service{}, &req)
 	assert.Error(t, err, "expected invalid uid and gid error, but received none")
 }
+
+func TestNewJailer_Isolated(t *testing.T) {
+	prepareIntegTest(t)
+
+	ociBundlePath, err := ioutil.TempDir("", t.Name())
+	require.NoError(t, err)
+	defer os.RemoveAll(ociBundlePath)
+	err = exec.Command("cp", "firecracker-runc-config.json.example", filepath.Join(ociBundlePath, "config.json")).Run()
```

Comment:
How about using CombinedOutput? If the cp doesn't work (e.g. disk is full), the err variable may not contain the information.

Reply:
That's fair. Will change.